src/battle_script_commands.c

```diff
@@ -11176,6 +11178,39 @@ static void Cmd_various(void)
         gBattlescriptCurrInstr = cmd->nextInstr;
         return;
     }
+    case VARIOUS_JUMP_IF_SHELL_TRAP:
```

Comment:
why is this included here?

Reply:
Because I moved it from its original position.

It was written above `VARIOUS_ACTIVATE_TERRAIN_CHANGE_ABILITIES`, even though it's defined below `VARIOUS_HIT_SWITCH_TARGET_FAILED`, so I moved it to keep consistency.